schedule/yast2_gui.yaml

```diff
@@ -6,13 +6,8 @@ description:    >
     as the gnome environment for the GUI tests.
 vars:
     BOOTFROM: c
-    BOOT_HDD_IMAGE: 1
-    # DESKTOP: gnome -> When added, an empty key appears first in the list
```

Comment:
I would not delete the comment if we still don't know how to fix this strange behavior.

Reply:
If we have some problem with this, we may create a ticket for the investigation. WDYT?:)

My opinion is that keeping commented stuff in production code may confuse other _hackers_ and may produce other commented lines in yaml files (which I guess we should avoid). I agree with @okurz on this matter: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7773#discussion_r299080684

Also, we should identify the list of variables that are used by scheduler and cannot be reached from yaml, and just do not include them here.

Reply:
I documented those two (I didn't remember it :) ), you're right, so let's remove them both.

Reply:
it might worth a ticket, but in my opinion we would need some tickets first to clean up variables before working in that one, as now it is kind of confusing where variables are coming from in some cases.